tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+
+from http import HTTPStatus
+
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from tcms.tests import LoggedInTestCase
+from tcms.tests.factories import UserFactory
+
+
+class TestSuperuserActions(LoggedInTestCase):
```

Comment:
The original class name was more or less fine, method names were not appropriate. Because you've made a change where you shouldn't have now you have to change it back. 

`TestTestRunAdmin` would be a suitable name b/c we are going to add all possible scenarios for TR admin here.